src/tools/miri/tests/fail/dangling_pointers/dangling_pointer_deref_underscore.rs

```diff
@@ -0,0 +1,11 @@
+// Make sure we find these even with many checks disabled.
+//@compile-flags: -Zmiri-disable-alignment-check -Zmiri-disable-stacked-borrows -Zmiri-disable-validation
+
+fn main() {
+    let p = {
+        let b = Box::new(42);
+        &*b as *const i32
+    };
+    let _ = unsafe { *p }; //~ ERROR: dereferenced after this allocation got freed
```

Comment:
```suggestion
    unsafe {
        let _ = *p; //~ ERROR: dereferenced after this allocation got freed
    }
```
Just to make sure the unsafe block doesn't interfere here...

Reply:
You are right. I forgot to implement the code to keep `PlaceMention` if requested... Added a commit for that.